src/ucp/proto/proto_multi.inl

```diff
@@ -44,9 +44,8 @@ ucp_proto_multi_scaled_length(uint32_t weight, size_t length)
 static UCS_F_ALWAYS_INLINE size_t
 ucp_proto_multi_max_payload(ucp_request_t *req,
                             const ucp_proto_multi_lane_priv_t *lpriv,
-                            size_t hdr_size)
+                            size_t hdr_size, size_t length)
```

Comment:
maybe pass hdr_size_first, hdr_size_middle?

Reply:
Reverted changes in `ucp_proto_multi_max_payload`.